README.md

```diff
@@ -21,6 +21,8 @@ Good luck.
 
 # EDIT_ME
 
+![devopslogo](/devopslogo.jpg)
```

Comment:
For some reason, it's still not previewing....
![Uploading CleanShot 2021-04-30 at 09.54.04.png…]()

Reply:
![CleanShot 2021-04-30 at 09 54 04](https://user-images.githubusercontent.com/323891/116639022-1ce8ab00-a99a-11eb-8361-55762ea53908.png)

Reply:
hmm that's odd... yesterday we managed to get it to work while on Allan's machine. @allandx could you try it again on your machine to see if it is still displaying?

Reply:
maybe remove the `/` 

can you also call it logo.jpg

Reply:
ok cool it works now! @allandx